### ISSUE TITLE
Difference between the two models on the results?

### ISSUE THREAD

**Issue by jeonsworld** [google-research/vision_transformer#21]
What is the difference between the first row and the second row of the table in expected result?
I expect the resulution size to be 224x224 or 384x384. Is that correct?

**Comment by andsteing** [comment]
I assume you refer to the first two rows in the table in the README?

| upstream    | model    | dataset      |   accuracy | wall_clock_time   | link                                                                                                                                                   |
|:------------|:---------|:-------------|-----------:|------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------|
| imagenet21k | ViT-B_16 | cifar10      |     0.9892 | 7.2h              | [tensorboard.dev](https://tensorboard.dev/experiment/KYu8HltjQdeGktGJyyQ6vw/#scalars&_smoothingWeight=0&regexInput=imagenet21k/ViT-B_16/cifar10/)      |
| imagenet21k | ViT-B_16 | cifar10      |     0.9903 | 7.7h              | [tensorboard.dev](https://tensorboard.dev/experiment/KYu8HltjQdeGktGJyyQ6vw/#scalars&_smoothingWeight=0&regexInput=imagenet21k/ViT-B_16/cifar10/)      |

There is no difference. It's simply the same configuration run two times to verify the first run was not an outlier.

**Comment by jeonsworld** [comment]
What is the resolution size used in the experiment from 224 or 384?
Thanks for your answer.

**Comment by andsteing** [comment]
All finetunings in that table use resolution 384 (this resolution is also used in the Colab) :

https://github.com/google-research/vision_transformer/blob/master/vit_jax/input_pipeline.py#L32-L54

Note that the models were pretrained on a resolution of 224 (for details please refer to the paper).

**Comment by jeonsworld** [comment]
Thanks!!
